MediaManager/Plugin.MediaManager.Android/MediaPlayerService.cs

```diff
@@ -353,11 +355,14 @@ private set
         /// <summary>
         /// Intializes the player.
         /// </summary>
-        public async Task Play(string url)
+        public async Task Play(string url, MediaFileType type)
```

Comment:
But this is a change to the interface, right? `public async Task Play(string url)` should only be an overload for developers convention